src/Xamarin.Android.Build.Tasks/Tasks/ResolveLibraryProjectImports.cs

```diff
@@ -83,10 +83,13 @@ public override bool Execute ()
 			var resolvedEnvironmentFiles      = new List<string> ();
 
 			assemblyMap.Load (AssemblyIdentityMapFile);
-			assembliesToSkip = new HashSet<string> (StringComparer.OrdinalIgnoreCase);
+			assembliesToSkipCaseFixup = new HashSet<string> (StringComparer.OrdinalIgnoreCase);
+			assembliestoSkipExtraction = new HashSet<string> (StringComparer.OrdinalIgnoreCase);
 			foreach (var asm in Assemblies) {
-				if (string.Compare (asm.GetMetadata (SkipAndroidResourceProcessing), bool.TrueString, StringComparison.OrdinalIgnoreCase) == 0)
-					assembliesToSkip.Add (asm.ItemSpec);
+				if (bool.TryParse (asm.GetMetadata (AndroidSkipResourceProcessing), out bool value) && value)
+					assembliesToSkipCaseFixup.Add (asm.ItemSpec);
+				if (bool.TryParse (asm.GetMetadata (GetAdditionalResourcesFromAssemblies.AndroidSkipResourceExtraction), out bool value) && value)
```

Comment:
This causes a similar failure: https://jenkins.mono-project.com/job/xamarin-android-pr-pipeline-release/652/console

```
error CS0128: A local variable or function named 'value' is already defined in this scope
```